bootstrap_cfn/fab_tasks.py

```diff
@@ -492,6 +492,18 @@ def get_zone_id():
                 "for zone name '%s'...", zone_id, zone_name)
     return zone_id
 
+def get_hosted_name(elb):
+    hosted_zone_name = elb.get('hosted_zone')
+    logger.info("fab_tasks::get_hosted_name: Found hosted zone name '%s' in config...",hosted_zone_name)
+    return hosted_zone_name
+
+
+def get_hosted_id(hosted_name):
+    r53_conn = get_connection(R53)
+    zone_id = r53_conn.get_hosted_zone_id(hosted_name)
+    logger.info("fab_tasks::get_hosted_id: Found hosted zone id '%s' in config...", zone_id)
+    return zone_id
+
```

Comment:
What if [1]:
```
def get_elb_details(elb):
   return (elb['name'], elb['hosted_zone'], ....get_hosted_zone_id(elb['hosted_zone']))
```
and skip get_hosted_id since it is already implemented (rework method: `get_zone_id` to get a parameter like zone_name=None so it is compatible.